R/alarm_add_plan.R

```diff
@@ -0,0 +1,25 @@
+#' Facilitates comparing an existing (i.e., non-simulated) redistricting plan to a set of simulated plans.
```

Comment:
This is good but we still need a title line above this.  See e.g. the docs for `alarm_50state_plans`.